manifests/cf-manifest/spec/manifest/rds_broker_spec.rb

```diff
@@ -75,78 +75,200 @@
-            expect(subject).to include(
+            expect(rds_properties).to include(
               "backup_retention_period" => 7
             )
           end
         end
 
+        shared_examples "backup disabled plans" do
+          it "calls out that it's not backed up in the description" do
+            expect(subject.fetch("description")).to include("NOT BACKED UP")
+          end
+
+          let(:rds_properties) { subject.fetch("rds_properties") }
+
+          it "has all snapshots disabled" do
+            expect(rds_properties).to include(
+              "backup_retention_period" => 0,
+              "skip_final_snapshot" => true,
+            )
+          end
+        end
+
+        shared_examples "HA plans" do
+          let(:rds_properties) { subject.fetch("rds_properties") }
+          it { expect(rds_properties).to include("multi_az" => true) }
+        end
+
+        shared_examples "non-HA plans" do
+          let(:rds_properties) { subject.fetch("rds_properties") }
+          it { expect(rds_properties).to include("multi_az" => false) }
+        end
+
+        shared_examples "Encryption disabled plans" do
+          let(:rds_properties) { subject.fetch("rds_properties") }
+          it { expect(rds_properties).to include("storage_encrypted" => false) }
+        end
+
+        shared_examples "Encryption enabled plans" do
+          let(:rds_properties) { subject.fetch("rds_properties") }
+          it { expect(rds_properties).to include("storage_encrypted" => true) }
+        end
+
+        describe "S-dedicated-9.5" do
+          let(:plan) { pg_plans.find { |p| p["name"] == "S-dedicated-9.5" } }
+          subject { plan }
+
+          it_behaves_like "small sized postgres plans"
+          it_behaves_like "backup enabled plans"
+          it_behaves_like "non-HA plans"
+          it_behaves_like "Encryption disabled plans"
+        end
+
+        describe "S-HA-dedicated-9.5" do
+          let(:plan) { pg_plans.find { |p| p["name"] == "S-HA-dedicated-9.5" } }
+          subject { plan }
+
+          it_behaves_like "small sized postgres plans"
+          it_behaves_like "backup enabled plans"
+          it_behaves_like "HA plans"
+          it_behaves_like "Encryption disabled plans"
+        end
+
         describe "M-dedicated-9.5" do
           let(:plan) { pg_plans.find { |p| p["name"] == "M-dedicated-9.5" } }
-          subject { plan.fetch("rds_properties") }
+          subject { plan }
 
           it_behaves_like "medium sized postgres plans"
-
-          it { is_expected.to include("multi_az" => false) }
+          it_behaves_like "backup enabled plans"
+          it_behaves_like "non-HA plans"
+          it_behaves_like "Encryption disabled plans"
         end
 
         describe "M-HA-dedicated-9.5" do
           let(:plan) { pg_plans.find { |p| p["name"] == "M-HA-dedicated-9.5" } }
-          subject { plan.fetch("rds_properties") }
+          subject { plan }
```

Comment:
With this refactoring, there's no real need to have a separate `let(:plan)` and `subject`. They could be combined into: `subject { pg_plans.find { |p| p["name"] == "M-HA-dedicated-9.5" } }`.